### ISSUE TITLE
[deploy cns] ansible ping output wrong

### ISSUE THREAD

**Issue by thoraxe** [openshift/openshift-cns-testdrive#254]
Output from the new 3.7 systems:

```
node01.internal.aws.testdrive.openshift.com | SUCCESS => {
    "changed": false, 
    "ping": "pong"
}
node03.internal.aws.testdrive.openshift.com | SUCCESS => {
    "changed": false, 
    "ping": "pong"
}
node02.internal.aws.testdrive.openshift.com | SUCCESS => {
    "changed": false, 
    "ping": "pong"
}
```

AWS account number is not present.

**Comment by thoraxe** [comment]
This is true in multiple places in the lab.

**Comment by dmesser** [comment]
solved by update 3.9
